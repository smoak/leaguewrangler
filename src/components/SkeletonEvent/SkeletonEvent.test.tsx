import React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';

import SkeletonEventWithStyles, { SkeletonEvent } from './SkeletonEvent';

describe('SkeletonEvent', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<SkeletonEvent classes={{ card: 'card' }} />);
  });

  it('renders as expected', () => {
    expect(component).toMatchSnapshot();
  });
});

describe('SkeletonEventWithStyles', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<SkeletonEventWithStyles />);
  });

  it('renders as expected', () => {
    expect(component).toMatchSnapshot();
  });
});
