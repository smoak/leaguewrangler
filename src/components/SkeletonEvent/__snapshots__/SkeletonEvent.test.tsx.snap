// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonEvent renders as expected 1`] = `
<WithStyles(ForwardRef(Card))
  className="card"
>
  <WithStyles(ForwardRef(CardHeader))
    avatar={
      <WithStyles(ForwardRef(Skeleton))
        height={40}
        variant="circle"
        width={40}
      />
    }
    subheader={
      <WithStyles(ForwardRef(Skeleton))
        height={6}
        width="80%"
      />
    }
    title={
      <WithStyles(ForwardRef(Skeleton))
        height={6}
      />
    }
  />
  <WithStyles(ForwardRef(CardContent))>
    <WithStyles(ForwardRef(Skeleton))
      height={50}
      variant="rect"
    />
  </WithStyles(ForwardRef(CardContent))>
  <WithStyles(ForwardRef(CardActions))>
    <WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Skeleton))
        variant="rect"
      />
    </WithStyles(ForwardRef(IconButton))>
  </WithStyles(ForwardRef(CardActions))>
</WithStyles(ForwardRef(Card))>
`;

exports[`SkeletonEventWithStyles renders as expected 1`] = `
<SkeletonEvent
  classes={
    Object {
      "card": "SkeletonEvent-card-1",
    }
  }
/>
`;
